test/CMakeLists.txt

```diff
@@ -12,3 +12,5 @@ add_dependencies(test_util gtest)
 add_subdirectory(common)
 add_subdirectory(storage)
 add_subdirectory(transaction)
+add_subdirectory(index)
```

Comment:
I think index will just live in storage, maybe as a subdirectory if we get enough files for that.

Reply:
ok :)